### DISCUSSION TITLE
What about between windows navigation.

### DISCUSSION THREAD

**Discussion by Uporka95** [dotnet/maui#13213] [Ideas]
Im develop desktop tray app via WPF and Im intersted in MAUI, but I dont have any idea how to implement navigation from one window to another like from page to page. 

There is Shell class which implies that app must have mainpage on the top of navigation stack. But in my case app's ui have several entry points to app's UI like dropzone, global keyboard handler (hotkeys) and tray, and there is no mainpage like in a mobile app or in others single window apps. 

Im looking for a way to navigate between dropzone/tray, settings menu, some overlays, and edtitors/previews.

Maybe Im missing something important or its really looks a bit far-fetched to try tranfer app with such type of architecture into MAUI framework.

**Comment by Xyncgas** [comment]
I did it because I implemented my own Rendering engine, but if you are saying between windows like browser window and visual studio window it's only possible via Windows api
